configs/nafnet/nafnet_c64eb11128mb1db1111_lr1e-3_400k_gopro.py

```diff
@@ -0,0 +1,126 @@
+_base_ = '../_base_/default_runtime.py'
+
```

Comment:
add the readme.md and readme_zh-cn.md under the configs

Reply:
Done